src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,308 @@
+import styled from '@emotion/styled';
+import { NextPage } from 'next';
+import Image from 'next/image';
+import React from 'react';
+import Background404 from '../../../../public/assets/img/background-404.png';
+import Background404Dark from '../../../../public/assets/img/background-dark-404.png';
+import BackgroundMobile404 from '../../../../public/assets/img/background-mobile-404.png';
+import BackgroundMobile404Dark from '../../../../public/assets/img/background-mobile-dark-404.png';
+import Logo404 from '../../../../public/assets/img/logo-404.png';
+import Logo404Dark from '../../../../public/assets/img/logo-dark-404.png';
+import { Typography, useMediaQuery } from '@mui/material';
+import lightTheme from '../../../../styles/theme/light';
+import { CustomButton } from '../custom-button/custom-button';
+import { useRouter } from 'next/router';
+import { useThemeContext } from '../../../core/context/ThemeContext';
+
+const CardNotFoundPage: NextPage = () => {
+  const router = useRouter();
+  const phoneLess = useMediaQuery(lightTheme.breakpoints.down('table_375'));
+  const isMobile = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const isLight = useThemeContext().themeMode === 'light';
+  const handleOnclick = () => {
+    router.push('/');
+  };
+  return (
+    <Wrapper>
+      <ImageContainer>
+        <Image
+          src={isLight && (phoneLess || isMobile) ? BackgroundMobile404 : !isLight && (phoneLess || isMobile) ? BackgroundMobile404Dark : isLight && !(phoneLess || isMobile) ? Background404 : Background404Dark}
```

Comment:
As this line is too big, maybe could make sense to format it this way (so it is more readable):

```js
isLight ? (
  // conditions for light theme
) : (
  // conditions for dark theme
)
```

Reply:
you are right, already done